teamworkApp/Gemfile

```diff
@@ -23,6 +23,8 @@ gem 'uglifier', '>= 1.3.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
 
+gem 'rubypython'
```

Comment:
Are you still using rubypython? It seemed like in the FB chat you got rid of it?